app/controllers/kudos_controller.rb

```diff
@@ -80,4 +81,8 @@ def can_add_another_kudo?
 
     redirect_to kudos_path, notice: 'You cannot add more kudos.'
   end
+
+  def pundit_user
+    current_employee
+  end
```

Comment:
I would suggest moving this to `ApplicationController`. Thanks to that we would inherit this method in all our controllers.

Reply:
Corrected ;) Thanks a lot 👍